libs/state/src/lib/rx-state.spec.ts

```diff
@@ -95,11 +95,86 @@ describe(rxState, () => {
     fixture.destroy();
     expect(RxState.prototype.ngOnDestroy).toHaveBeenCalled();
   });
+
+  describe('signals', () => {
+    it('should be undefined when key is undefined', () => {
+      const { component } = setupComponent<{ count: number }>();
+      const state = component.state;
+      const count = state.signal('count');
+
+      expect(count()).toBe(undefined);
```

Comment:
Does the type also include undefined?

Reply:
Yes it is, because signals need an initial value.

Reply:
the initial value can be `undefined` in case the state has no value for that key. Same is true when calling `get`, yet it's not part of the type.